### PR TITLE
request information for a windfarm feature collection

### PR DESCRIPTION
**Pull request by SiggyF**
Get wind and bathymetry  information for a  feature collection

### CODE REVIEW

File: hydroengine_service/main.py [hunk 1]
```diff
@@ -1470,6 +1474,75 @@ def get_metocean_data():
...
+@app.route('/get_windfarm_data', methods=['POST'])
+@flask_cors.cross_origin()
+def get_windfarm_data():
+    r = request.get_json()
+
+    features = r['features']
+    collection = ee.FeatureCollection(features)
+
+
+    ports = ee.FeatureCollection("users/fbaart/worldlogistic/port")
```
Comment (gena) [inline hydroengine_service/main.py:1486]: move this to projects/dgds-gee

File: hydroengine_service/main.py [hunk 2]
```diff
@@ -1470,6 +1474,75 @@ def get_metocean_data():
...
+    gebco = ee.Image("projects/dgds-gee/gebco/2019")
+    coast = ee.Image("users/gena/land_polygons_image")
```
Comment (gena) [inline hydroengine_service/main.py:1488]: move this to projects/dgds-gee

File: hydroengine_service/main.py [hunk 3]
```diff
@@ -1470,6 +1474,75 @@ def get_metocean_data():
...
+    wind = ee.Image("projects/dgds-gee/gwa/gwa3/10m").rename('wind_magnitude_mean')
+
+    scale = 1000
```
Comment (gena) [inline hydroengine_service/main.py:1491]: fixing scale might be an issue, might be better to pass as request parameters

Comment (SiggyF) [reply hydroengine_service/main.py:1491]: I want this specific at this scale.

File: hydroengine_service/main.py [hunk 4]
```diff
@@ -1506,6 +1510,75 @@ def get_metocean_data():
...
+@app.route('/get_windfarm_data', methods=['POST'])
+@flask_cors.cross_origin()
+def get_windfarm_data():
+    r = request.get_json()
+
+    features = r['features']
+    collection = ee.FeatureCollection(features)
+
+
+    ports = ee.FeatureCollection("users/fbaart/worldlogistic/port")
```
Comment (SiggyF) [inline hydroengine_service/main.py:1522]:
```suggestion
    ports = ee.FeatureCollection("projects/dgds-gee/worldlogistic/port")
```

File: hydroengine_service/main.py [hunk 5]
```diff
@@ -1506,6 +1510,75 @@ def get_metocean_data():
...
+def get_windfarm_data():
+    r = request.get_json()
+
+    features = r['features']
+    collection = ee.FeatureCollection(features)
+
+
+    ports = ee.FeatureCollection("projects/dgds-gee/worldlogistic/port")
+    gebco = ee.Image("projects/dgds-gee/gebco/2019")
+    coast = ee.Image("users/gena/land_polygons_image")
```
Comment (SiggyF) [inline hydroengine_service/main.py:1524]:
```suggestion
    coast = ee.Image("projects/dgds-gee/osm/land_polygons_image")
```

**Review by SiggyF — COMMENTED** [review]
updated dataset

File: hydroengine_service/main.py (hunk 5 above)
Comment (SiggyF) [reply hydroengine_service/main.py]: moved

File: hydroengine_service/main.py (hunk 1 above)
Comment (SiggyF) [reply hydroengine_service/main.py]: moved

**Review by gena — APPROVED** [review]

**Pull request merged**